### PR TITLE
Local Assistant man can now bash rock again!*

### PR DESCRIPTION
**Pull request by TheObserver-sys**
## About The Pull Request
*only on maps commonly voted in rotation, as well as Lambda.

This restores the public mining shuttle on Box, Delta, Lambda, Meta, and Pubby, as well as public mining section on the mining station.
A massive thank you to Acer202 for actually knowing how to map properly, because boy howdy I don't.
This also fixes a runtime due to the monastery shuttle not exactly liking lavaland, by just making a new shuttle for Pubby.

## Why It's Good For The Game
It's mixed more than anything. On one hand, this allows botanists and chemists to chase down rare plants and chemicals freely in their downtime, as well as allow local assistant man to work in the coal mine to buy their precious gloves, and allows robos to just go mining themselves if no miners exist. On the other hand, Lavaland will eternally be a balancing nightmare, something I can't be bothered to try and fix due to the overwhelming negative reaction of touching any loot from Lavaland, even if it'd deserve nerfs. If this goes through, I'll be reviewing a few of the mutations of plants elsewhere and potentially adding them here as well. 

There are other merits to allowing the crew a more regular amount of access to lavaland, part of that being the ability to actually obtain a decent anvil for forging, a large open space for experimentation, should the want arise, and being able to actually bust illicit gatherings of likeminded people with grenades and stun batons in a reasonable timeframe.

## Changelog
:cl: Acer202, with minor help from The0bserver
add: After internal deliberation, CentCom has decided to run a limited reinstatement of public mining shuttles for use in more tried and true station classes. CentCom would like to remind you that this privilege is easily revoked, and that abuse may result in immediate detonation.
add: Restores the mining shuttle on Pubby, Box, Delta, Meta, and Lambda Station.
/:cl:

### CODE REVIEW

**Review by silicons — DISMISSED** [review]

**Comment by MrJWhit** [comment]
Do not re-add public mining. 

1. All it does is create tension between miners and everyone else, as people will **instantly** break into mining to get their shit and fuck up what they're doing.
2. Lavaland isn't balanced in the slightest to give everyone access to it (as seen last time), saying you'll fix it later doesn't mean anything if you don't touch the actual weapons that make lavaland so damn good. Plants be damned, it's the fact that you can get a katana and better weapons than anything else.
3. People running away to lavaland makes hunting them down as an antag 100x harder than anything else in the entire game. Someone just popping down, and going to a forgotten corner to ERP is something that happens, and there's nothing you can do about it as an antag and makes it impossible to actually do anything.
4. Non-miners can't even access to the ORM to drop off the minerals they pick up, it just exists so people can go pick up gamer gear.

**Comment by DeltaFire15** [comment]
All the previous issues, save for legion cores being aheals (and instead being somewhat weaker now) still exist, just to name two,
people using the mining dorms as dorms++ to not get targetted by any antags aswell as greytiders rushing all the legion tendrils for gamer gear and not actually mining a single bit of ore.

If you really want mining access for your projects or to help out, you can ask the HoP for it, considering nothing has been significantly changed about the state of mining between public mining being removed and this PR

**Comment by moltoretardo** [comment]
For a temporary solution against low pop I think it's neat. On high pop it was kinda bad for the miners but on a low pop server when there's no miners or hop to give you the access it can be real good. Maybe add an ORM for the assistant miners so they can atleast put the gathered ores somewhere. Maybe a nerfed shop for them too or something. Also maybe make the shuttle accessible only when there's 0 miners on the manifest so it solves the apprentices miner outdoing the miners problem.

For the "powergaming antags or greytiders that aren't miners even if miners always do it without problems" just I don't know, warn players not to do it? Add a rule that state : "Don't be a dick" so that the people that can read knows that it wouldn't be fun to powergame and fuck up a station with overpowered gear on a lowpop round...maybe ban the one that does it? Hmmmm I unno.

People going into forgotten place to erp was always a problem and I don't think it's a rather big one. "Boohoo my greentext" yeah it's not like anyone really cared about it eitherways. Gosh darn it my target isn't on station which mean the potential of me doing a fun gimmick is ruined because people won't see me getting that epic **greentext**. Just remove antags objectives or ban erp \s

I think making a maybe less potent but still accessible ORM for the non-miner could be good. One that can't dispenses ores (even if every techfab can just do it so people don't scream powergame) and gives maybe reducted points. Something like that.

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -79,6 +79,15 @@
 	no_destination_swap = TRUE
 	var/static/list/dumb_rev_heads = list()
 
+	/obj/machinery/computer/shuttle/mining/common
```
Comment (Trilbyspaceclone) [inline code/modules/mining/mine_items.dm:82]:
```suggestion
/obj/machinery/computer/shuttle/mining/common
```

**Comment by TheObserver-sys** [comment]
Alright, looks like I need to call on some of this.

1. The literal only difference in a public mining world between a miner and an assistant miner is a duffle bag with the goodboy access card. Honestly, in the current state of the server, I would trust assistants more than I would trust a traditional miner to bring the station back a good load of materials. As for breaking in, we currently still have to do that in a pre-public mining world in the event all of our miners eat shit, and we need to direct paramedics to search for them, be it through breaking into teleporter, or breaking all the way into cargo, which will still piss people off. Also, we still have a pain of a time even doing that because the mining shuttle is ID locked, so the shuttle may very well be trapped down there with no way for us to get access to it except through breaking into teleporter.

2. Not gonna lie chief, while I know this, I don't hear anyone having any actual ideas to balance it to not be a pure gamer loot cave, outside killing off lavaland entirely, and returning to asteroid mining, That decision would honestly piss people off far more than restoring public mining, considering people look forward to punching large monsters in the dick.  Perhaps we should take a hard look at chest and megafauna loot at some point, and plug our ears with concrete  at the outcry that will follow, but that is _not_ the concern of this PR.

3. I won't deny that it does make it a hell of a lot harder, but by the same token, you know what makes it hard to hunt down a target? Any snow map, or anyone who can get access to a space suit, yet we don't see PR's that reinforce EVA to the point of absurdity just to prevent people from running away from the station. Hell, we deadass give the Curator one for free, and just go _'have a nice trip!'_ As for ERPing in the mining dorms, if you are extremely confident that your target is in there, you can always attempt a break in/ blow a hole into it and kill them, with their partner as the only collateral.

4. This is true on a whole **one map**, said map being Box Station, and only because we've changed the design of it to nestle it by mining instead of where it is on every other codebase.  On every other map, an assistant can just wrench the ORM out, slide their box against the back of it, shove it back in, and wrench it back down, none the wiser.

5. Asking for a HoP/ HoPlike entity to change your ID is depending on there being any command to speak of. This also depends on the sole command officer to be the kinda person to not just run down to Lavaland with the spare, and then **die down there**. You could say, _'just greytide in! it'll be fine!"_, but that is contingent upon who is in the Adminwho tab on the server, or an antagonist with a heart of passable gold substitute to run the station while also stealing/killing who they need to.

**Pull request merged**